lib/Distribution/Sle/16Latest.pm

```diff
@@ -0,0 +1,24 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: The class represents current (i.e. latest) SLE 16 distribution and
+# provides access to its features.
+
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+package Distribution::Sle::16Latest;
+use strict;
+use warnings FATAL => 'all';
+use parent Distribution::Opensuse::AgamaDevel;
+
+use Yam::Agama::Pom::GrubMenuPage;
+use Yam::Agama::Pom::GrubEntryEditionPage;
+use Yam::Agama::Pom::AgamaUpAndRunningPage;
+use Yam::Agama::Pom::RebootPage;
+use Yam::Agama::Pom::RebootTextmodePage;
+
+use Utils::Architectures;
```

Comment:
you don't need any of this either....

Reply:
removed, a VR test is running https://openqa.opensuse.org/tests/4514637#live